apps/playground-web/services/webServices.ts

```diff
@@ -41,19 +41,15 @@ export const WebService = {
     try {
       const response = await fetch(`${PLAYGROUND_MONO_URL}${url}`, options);
 
-      // If the response is not OK, check if the response contains error information
-      if (!response.ok) {
-        const errorResponse = await response.json();
-        if (errorResponse?.error) {
-          throw errorResponse.error;
-        } else {
-          throw new Error(`HTTP error! Status: ${response.status}`);
-        }
-      }
+      const headers = {};
+      response.headers.forEach((value, key) => {
+          headers[key] = value;
+      });
+      headers['x-ratelimit-remaining'] = 101; 
```

Comment:
Why're we hardcoding this number?

Reply:
This was just a temporary change, I'll remove it. It looks like chrome blocks ceratin response headers to be read by the react app, so I had to hardcode this to test it out.

Reply:
I've removed this change and done this corresponding change in playground-mono: https://github.com/DiceDB/playground-mono/pull/41/files